README.rst

```diff
@@ -63,11 +63,11 @@ connection, but historically has been over PS/2 or ADB connections.
 
 *In the case of Virtual Keyboard (as in touch screen devices):*
 
-- In modern capacitive touch screens when the user puts their finger on the
-  screen a tiny amount of current from the electrostatic field of the
-  conductive layer gets transferred to the finger completing the circuit
-  and creating a voltage dropping at that point on the screen so that the
-  ``screen controller`` raises an interrupt reporting the coordinate of
+- In modern capacitive touch screens; when the user puts their finger on the
```

Comment:
I don't think this is correct. The semicolon separates independent clauses, but

> In modern capacitive touch screens

isn't an independent clause. How about

> When the user puts their finger on a modern capacitive touch screen…